src/components/field-label/field-label.visualroute.js

```diff
@@ -58,5 +58,8 @@ export const component = () => (
         horizontalConstraint="m"
       />
     </Spec>
+    <Spec label="with inverted tone">
```

Comment:
you can access the `themes` prop that is passed in here.
```js
export const component = ({ themes }) => (
```

and then you can do 

```js

      <ThemeProvider theme={themes.darkTheme}>
        <Spec label="with inverted tone">
//
        </Spec>
      </ThemeProvider>

```

Reply:
hm, maybe this won't really work because now the theme will apply to the "Text" inside of your component. In that case you would need to use ThemeProvider twice 🤔

Reply:
Let's see. Two commits incoming.